templates/classroom-single.php

```diff
@@ -0,0 +1,12 @@
+<?php get_header();
+
+$classroom = $args['classroom'];
+
+dd( $classroom );
```

Comment:
`dd()` is generally a debug function, is this intended to be left here (perhaps `dd()` in this project is something different and I've missed it). If it is intended to output some content, please can this be escaped.